enterprise/frontend/src/embedding-sdk/README.md

```diff
@@ -1189,6 +1189,169 @@ async function fetchRequestToken(url) {
 const config = {fetchRequestToken};
 ```
 
+### Using with Next.js
+
+#### Using App Router
+
+Create a component that imports the `MetabaseProvider` and mark it a React Client component with "use client";
+
+```typescript jsx
+"use client";
+
+import { MetabaseProvider, StaticQuestion } from "@metabase/embedding-sdk-react";
+
+const config = {...}; // Your Metabase SDK configuration
+
+export default function MetabaseComponents() {
+  return (
+    <MetabaseProvider config={config}>
+      <StaticQuestion questionId={QUESTION_ID} />
+    </MetabaseProvider>
+  );
+```
+
+Make sure to use default export, as named export is not supported with this setup.
+
+Then, import this component in your page:
+
+```typescript jsx
+// page.tsx
+
+const MetabaseComponentsNoSsr = dynamic(() => import("@/components/MetabaseComponents"), {
+  ssr: false
+});
+
+export default function HomePage() {
+  return (
+    <>
+      <MetabaseComponentsNoSsr />
+    </>
+  );
+}
+```
+
+> [!CAUTION]
+> If you export the component as a named export, it will not work with Next.js. You must use a default export.
+
+This won't work:
+
+```typescript jsx
+const DynamicAnalytics = dynamic(
+  () => import("@/components/MetabaseComponents").then((module) => module.MetabaseComponents),
+  {
+    ssr: false,
+  }
+);
+```
+
+If you authenticate with Metabase using JWT, you can create an a Route handler that sign a user into Metabase.
```

Comment:
```suggestion
If you authenticate with Metabase using JWT, you can create a Route handler that signs a user into Metabase.
```

Reply:
Fixed in 29c628df0652b29bb21ca46cdbf953f8626fc26a